### DISCUSSION TITLE
Is it possible to use cypress-firebase in combination with the version 9 modular of firebase?

### DISCUSSION THREAD

**Discussion by haxolotl** [prescottprue/cypress-firebase#805] [Q&A]
After upgrading to firebase 9, I'm trying to get it working for my cypress setup. I have read through the readme, and found the section that mentions version 9 of firebase, but it only seems to be supporting the compat, rather than modular version 9. Is this something that is possible now? Or will support be added for it at any point in the future?

Hope I didn't miss this information anywhere else!
 Thanks :)

**Comment by Tockra** [comment]
I'm a little bit surprised. It is possible and its documented. The current version is buggy, but version 2.2.5 works with firebase 9
